components/climate/pid.rst

```diff
@@ -54,6 +59,35 @@ Configuration variables:
   - **max_integral** (*Optional*, float): The minimum value of the integral term multiplied by
     ``ki`` to prevent windup. Defaults to ``1``.
 
+  - **output_samples** (*Optional*, int): average the output over this many samples. PID controllers 
+    can be quite sensitive to small changes on the input sensor. By averaging the last X output samples,  
+    the temperature can be more stable. However, the larger the sampling window, the less responsive the 
+    PID controller. Defaults to ``1`` which is no sampling/averaging.
+
+  - **derivative_samples** (*Optional*, int): average the derivative term over this many samples. Many 
+    controllers don't use the derivative term because it is sensitive to slight changes in the input sensor. 
+    By taking an average of the derivative term it might become more useful for you. Most PID controllers call 
+    this derivative filtering. The derivative term is used to pre-act so don't filter too much. Defaults to ``1`` 
+    which is no sampling/averaging.
+
+- **deadband_parameters** (**Optional**): Enables a deadband to stabilise and minimise changes in the 
```

Comment:
GitHub won't let me add suggestions for some reason, but please remove one pair of asterisks around the Optional (double only for required keys)